x-pack/plugins/apm/public/components/app/service_overview/service_overview_dependencies_table/index.tsx

```diff
@@ -96,7 +96,7 @@ export function ServiceOverviewDependenciesTable({ serviceName }: Props) {
       name: i18n.translate(
         'xpack.apm.serviceOverview.dependenciesTableColumnLatency',
         {
-          defaultMessage: 'Latency',
+          defaultMessage: 'Latency (avg.)',
```

Comment:
Can't we also use `getLatencyColumnLabel` here?

Reply:
We could do `getLatencyColumnLabel(undefined)` but it's always going to be avg because that's the only thing we can get with the dependencies query.